app/views/users/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<ul>
+  <% @users.each do |user| %>
+    <li>
+      <a href="<%= user_posts_path(user) %>">
```

Comment:
- [ ] Please fix the route on your user index page and make it redirect to the user show page. Example snippet:

```rb
<a href= "/users/<%=user.id%>">
```

Reply:
This change has done well